src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -1128,6 +1129,7 @@ public void test100PercentContaminationNoCallsInGVCFMode() throws Exception {
                 "-contamination", "1.0",
                 "--" + AssemblyBasedCallerArgumentCollection.EMIT_REF_CONFIDENCE_LONG_NAME, ReferenceConfidenceMode.GVCF.toString(),
                 "--" + AssemblyBasedCallerArgumentCollection.ALLELE_EXTENSION_LONG_NAME, "2",
+                "--" + StandardArgumentDefinitions.VERBOSITY_NAME, Log.LogLevel.WARNING.toString()  //shows DP and SB warnings
```

Comment:
I'm not sure there's any point to piggybacking on these existing tests to "cover" this warning message, since it affects only the logging and not the output. It would be better, I think, to write a 2-3 line unit test for the new `generateMissingDataWarning()` method that just exercises the two cases of `likelihoods == null` and `! g.isCalled()`

Reply:
Well, it certainly made me feel better to see it in the console output.